README.md

```diff
@@ -7,17 +7,22 @@ Code and documentation for QSAR modeling and virtual screening of small molecule
 3. Clone and enter this repo by running
 ```shell script
 git clone --recurse-submodules https://github.com/srensi/tmprss2.git
+# (advanced) If you are authenticating GitHub with ssh, use this line instead
+git clone --recurse-submodules git@github.com:srensi/tmprss2.git
 cd tmprss2
 ```
 4. Install dependencies by running these from the tmprss2 directory:
 ```shell script
-#Install dependencies:
+# Install dependencies:
 conda env create -f environment.yml
-conda env update -n tmprss2 -f chemprop_fitting/chemprop_repo/environment.yml
 conda activate tmprss2
 
 # Install chemprop as package
 pip install -e chemprop_fitting/chemprop_repo
+
+# Install pubchempy
+pip install pubchempy
```

Comment:
You can add this to the environment.yml file under the pip packages, e.g.
pip:
     - pubchempy
that should work

Reply:
added, sweet did not know this could be done